x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -14,13 +14,41 @@
 import java.util.Map;
 
 public class EsqlFeatures implements FeatureSpecification {
-    private static final NodeFeature MV_LOAD = new NodeFeature("esql.mv_load");
+    /**
+     * When we added the warnings for multivalued fields emitting {@code null}
+     * when they touched multivalued fields.
+     */
+    private static final NodeFeature MV_WARN = new NodeFeature("esql.mv_warn");
```

Comment:
Maybe mention the version in the javadoc for each feature (and potentially the PR/issue that added it) it's specified in the map but might help navigate this in X months from now, trying to diagnose some bug.